magma_access_gateway_installer/__init__.py

```diff
@@ -69,18 +72,73 @@ def cli_arguments_parser(cli_arguments):
         "--ip-address",
         dest="ip_address",
         required=False,
-        help="Statically allocated SGi interface IP address. Example: 1.1.1.1/24",
+        help="Statically allocated SGi interface IP address. Example: 1.1.1.1/24.",
     )
     cli_options.add_argument(
         "--gw-ip-address",
         dest="gw_ip_address",
         required=False,
-        help="Upstream router IP for SGi interface. Example: 1.1.1.200",
+        help="Upstream router IP for SGi interface. Example: 1.1.1.200.",
+    )
+    cli_options.add_argument(
+        "--skip-networking",
+        dest="skip_networking",
+        action="store_true",
+        required=False,
+        help="If used, network configuration part of the installation process will be skipped. "
+        "In this case operator is responsible for providing expected network configuration.",
```

Comment:
Is that space requiered?
```suggestion
        help="If used, network configuration part of the installation process will be skipped."
        "In this case operator is responsible for providing expected network configuration.",
```

Reply:
It is. It's a split string to avoid exceeding max number of chars in the line. When printed to the console, it's gonna be one line.